### PR TITLE
Superlu_dist 4.1 (updated)

### PR DESCRIPTION
**Pull request by davydden**

### CODE REVIEW

File: superlu_dist.rb [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
-  url "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_3.3.tar.gz"
-  sha256 "d2fd8dc847ae63ed7980cff2ad4db8d117640ecdf0234c9711e0f6ee1398cac2"
-  revision 2
-
-  bottle do
-    root_url "https://homebrew.bintray.com/bottles-science"
-    sha256 "4ee905552a04328f91b8ed94ef5de50e6528ee9901d0809fca67cf3bfaa759cc" => :yosemite
-    sha256 "07e5ccc50442e335e68a2787166f138b1d1f9ee492ad99153cd29de793a0179e" => :mavericks
-    sha256 "ec296ebdd3e9af551c99969953fb1390c0d516cc9ba9a56d0e2097a85b06f85c" => :mountain_lion
-  end
```
Comment (dpo) [inline superlu_dist.rb]: Don't remove the bottle section.

**Comment by dpo** [comment]
```
==> Downloading http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_4.0.tar.gz
/usr/bin/curl -fLA Homebrew 0.9.5 (Ruby 2.0.0-481; OS X 10.9.5) http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_4.0.tar.gz -C 0 -o /Library/Caches/Homebrew/superlu_dist-4.0.tar.gz.incomplete --silent
Error: Failed to download resource "superlu_dist"
Download failed: http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_4.0.tar.gz
```

**Comment by davydden** [comment]
@dpo their website is down now. I don't think it is a mistake in url

**Comment by dpo** [comment]
Still seems to be down. I contacted the author.

**Comment by davydden** [comment]
it's up again.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
There is an error in one of the tests:
```
ldb must be >= MAX(K,1): ldb=0 K=0BLAS error: Parameter number 11 passed to cblas_dgemm had an invalid value
```

**Comment by davydden** [comment]
i wonde why it wasn't picked up in 
```
$ brew test superlu_dist
Testing davydden/science/superlu_dist
==> mpirun -np 4 ./pddrive -r 2 -c 2 g20.rua
==> mpirun -np 10 ./pddrive4 g20.rua
==> mpirun -np 4 ./pzdrive -r 2 -c 2 cg20.cua
==> mpirun -np 10 ./pzdrive4 cg20.cua
```
hm...

I am aware of this error, but i hoped it appears in Superlu_dist compiled by PETSc only. *SIGHS*
Apparently it isn't.

The authors know about it (i reported it in PETSc forum, Barry Smith forwarded it to the authors), it seems to be the for linux as well. 

I propose to keep this PR and Trilinos open, and merge PETSc without waiting for this one even though it does compile with 4.0 as well.

**Comment by dpo** [comment]
Ok thanks!

**Comment by davydden** [comment]
tests pass now

File: superlu_dist.rb [hunk 2]
```diff
@@ -65,3 +71,4 @@ def install
     end
   end
 end
+
```
Comment (dpo) [inline superlu_dist.rb]: Please remove extra blank line.

**Comment by dpo** [comment]
Thank you!

**Pull request closed without merging**